robocop/checkers/naming.py

```diff
@@ -676,40 +641,22 @@ def find_similar_variables(self, tokens, node):
 class DeprecatedStatementChecker(VisitorChecker):
     """Checker for deprecated statements."""
 
-    reports = (
-        "deprecated-statement",
-    )
+    reports = ("deprecated-statement",)
     # deprecated:alternative
     deprecated_statements_rf3 = {}
     deprecated_statements_rf4 = {"runkeywordunless": "IF"}
-    deprecated_statements_rf5 = {"runkeywordunless": "IF",
-                                 "runkeywordif":     "IF"
-                                 }
-
-    def visit_SuiteSetup(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_TestSetup(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
+    deprecated_statements_rf5 = {"runkeywordunless": "IF", "runkeywordif": "IF"}
 
     def visit_Setup(self, node):  # noqa
         self.check_if_keyword_is_deprecated(node.name, node)
 
-    def visit_SuiteTeardown(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_TestTeardown(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
-
-    def visit_Teardown(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.name, node)
+    visit_KeywordCall = (
+        visit_TestTeardown
+    ) = visit_SuiteTeardown = visit_Teardown = visit_TestSetup = visit_SuiteSetup = visit_Setup
 
     def visit_TestCase(self, node):  # noqa
         self.generic_visit(node)
 
-    def visit_KeywordCall(self, node):  # noqa
-        self.check_if_keyword_is_deprecated(node.keyword, node)
```

Comment:
visit_KeywordCall and let's say visit_SuiteTeardown have different implementation - the first uses node.keyword and the other node.name. But that should be covered by me in the other PR (#576) already

Reply:
I will merge this PR only after yours is merged. Then I will synchronize them.